src/onObject/references.ts

```diff
@@ -0,0 +1,137 @@
+import powerbi from "powerbi-visuals-api";
+import SubSelectableDirectEdit = powerbi.visuals.SubSelectableDirectEdit;
+import SubSelectableDirectEditStyle = powerbi.visuals.SubSelectableDirectEditStyle;
+
+import { IFontReference, ILegendReference, ILabelsReference, IDataPointReference, IDisplayReference, ILineReference, IYAxisLabelsReference } from "./interfaces";
+import { RadarChartObjectNames } from "../settings";
+
+export const TitleEdit: SubSelectableDirectEdit = {
+    reference: {
+        objectName: RadarChartObjectNames.Legend,
+        propertyName: "titleText"
+    },
+    style: SubSelectableDirectEditStyle.HorizontalLeft,
+}
+export const visualTitleEditSubSelection = JSON.stringify(TitleEdit);
+
+const createBaseFontReference = (objectName: string, colorName: string, prefix: string = ""): IFontReference => {
+    return {
+        fontFamily: {
+            objectName: objectName,
+            propertyName: `${prefix}fontFamily`
+        },
+        bold: {
+            objectName: objectName,
+            propertyName: `${prefix}fontBold`
+        },
+        italic: {
+            objectName: objectName,
+            propertyName: `${prefix}fontItalic`
+        },
+        underline: {
+            objectName: objectName,
+            propertyName: `${prefix}fontUnderline`
+        },
+        fontSize: {
+            objectName: objectName,
+            propertyName: `${prefix}fontSize`
+        },
+        color: {
+            objectName: objectName,
+            propertyName: colorName
+        }
+    }
+}
+
+export const legendReferences: ILegendReference = {
+    ...createBaseFontReference(RadarChartObjectNames.Legend, "labelColor"),
+    cardUid: "Visual-legend-card",
+    groupUid: "legendTextGroup-group",
+    show: {
+        objectName: RadarChartObjectNames.Legend,
+        propertyName: "show"
+    },
+    showTitle: {
+        objectName: RadarChartObjectNames.Legend,
+        propertyName: "showTitle"
+    },
+    titleText: {
+        objectName: RadarChartObjectNames.Legend,
+        propertyName: "titleText"
+    },
+    position: {
+        objectName: RadarChartObjectNames.Legend,
+        propertyName: "position"
+    },
+    color: {
```

Comment:
aren't color property is already correctly defined from `...createBaseFontReference`?